src/tabs/presets/PresetsDetailedDialog.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+class PresetsDetailedDialog
+{
+    constructor(domDialog, cliEngine, presetsRepo)
+    {
+        this._domDialog = domDialog;
+        this._cliEngine = cliEngine;
+        this._presetsRepo = presetsRepo;
+    }
+
+    load(onLoadedCallback)
+    {
+        this._onLoadedCallback = onLoadedCallback;
+        this._domDialog.load("./tabs/presets/PresetsDetailedDialog.html", () =>
+        {
+            this._setupdialog();
+        });
+    }
+
+    open(preset)
+    {
+        this._preset = preset;
+        this._updatePresetUi();
+        this._domDialog[0].showModal();
+    }
+
+    _updatePresetUi()
+    {
+        this._domTitle.text(this._preset.title);
+        this._domAuthor.text(this._preset.author);
+        this._domKeywords.text(this._preset.keywords?.join("; "));
+        this._domVersions.text(this._preset.firmwareVersion?.join("; "));
+        this._domCategory.text(this._preset.category);
+        this._domOfficial.text(this._preset.official);
+        this._domDescription.html(this._preset.description?.join("<br/>"));
+
+        this._domGitHubLink.attr("href", this._presetsRepo.getPresetGitHubLink(this._preset));
+
+        this._setLoadingState(false);
+    }
+
+    _setLoadingState(isLoading)
+    {
+        this._domProperties.toggle(!isLoading);
+        this._domLoading.toggle(isLoading);
+        this._domButtonApply.toggle(!isLoading);
+    }
+
+    _readDom()
+    {
+        this._domButtonApply = $('#presets_detailed_dialog_applybtn');
+        this._domButtonCancel = $('#presets_detailed_dialog_closebtn');
+        this._domLoading = $('#presets_detailed_dialog_loading');
+        this._domProperties = $('#presets_detailed_dialog_properties');
+
+        this._domTitle = this._domDialog.find('.preset_detailed_dialog_title');
+        this._domAuthor = this._domDialog.find('.preset_detailed_dialog_author');
+        this._domKeywords = this._domDialog.find('.preset_detailed_dialog_keywords');
+        this._domVersions = this._domDialog.find('.preset_detailed_dialog_vesions');
+        this._domCategory = this._domDialog.find('.preset_detailed_dialog_category');
+        this._domOfficial = this._domDialog.find('.preset_detailed_dialog_official');
+        this._domDescription = this._domDialog.find('.preset_detailed_dialog_description');
+        this._domGitHubLink = this._domDialog.find('#presets_github_link');
+    }
+
+    _setupdialog()
+    {
+        i18n.localizePage();
+        this._readDom();
+
+        this._domButtonApply.on("click", () =>
```

Comment:
Could be a oneliner:
```suggestion
        this._domButtonApply.on("click", () => this._onApplyButtonClicked());
```